apps/prairielearn/src/tests/activeAccessRestriction.test.ts

```diff
@@ -25,8 +25,7 @@ describe('Exam and homework assessment with active access restriction', function
 
   const headers: Record<string, string> = {};
 
-  const VARIANT_FORBIDDEN_STRING =
-    'This question was not viewed while the assessment was open, so no variant was created.';
+  const VARIANT_FORBIDDEN_STRING = 'This question was not viewed while the assessment was open';
```

Comment:
From what I can tell, Cheerio's `:contains(...)` selector doesn't correctly handle text that's wrapped across multiple lines in the raw HTML. We don't have control over how Prettier formats the HTML, so the next best thing was just just check for a smaller bit of text that's currently all on one line.

If we have to change this again in the future, it might be better to check for a `data-testid` or the like.